src/Tribe/Settings.php

```diff
@@ -613,10 +613,6 @@ public function get_url( array $args = array() ) {
 			$args = wp_parse_args( $args, $defaults );
 			$parent = self::$parent_page;
 
-			if ( ! is_admin() ) {
```

Comment:
Just asking to be sure we're covered: but removing this won't cause a regression re [PR 95](https://github.com/moderntribe/tribe-common/pull/95) / [issue #46198](https://central.tri.be/issues/46198)?